dashboards-observability/public/components/explorer/visualizations/config_panel/config_editor/config_controls/config_thresholds.tsx

```diff
@@ -102,43 +103,53 @@ export const ConfigThresholds = ({
               <EuiFormRow fullWidth label="">
                 <EuiFlexGroup alignItems="center" gutterSize="xs">
                   <EuiFlexItem grow={3}>
-                    <EuiColorPicker
-                      fullWidth
-                      onChange={handleThresholdChange(thr.thid, 'color')}
-                      color={thr.color}
-                    />
+                    <EuiFormRow helpText="color">
```

Comment:
Is wrapping them in `EuiFormRow` just to add the helpText or does it change the look as well??

Reply:
No look change, just add helper text, but having one side effect is all the input elements get lifted up a little bit, but wrapping them all achieved no look change